<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuseumPage</class>
 <widget class="QDialog" name="MuseumPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="muName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next Condensed</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;h1&gt;Museum Name&lt;/h1&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoMM">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>0</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>791</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="homeButton">
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mListButton">
      <property name="text">
       <string>Museum list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="itemsViewedButton">
      <property name="text">
       <string>Items Viewed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="myCommentsButton">
      <property name="text">
       <string>My Comments</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="accountButton">
      <property name="text">
       <string>Account</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="image1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Image 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="image4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Image 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="image7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Image 7</string>
   </property>
  </widget>
  <widget class="QLabel" name="image2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Image 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="image3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Image 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="image5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Image 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="image8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Image 8</string>
   </property>
  </widget>
  <widget class="QLabel" name="image6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Image 6</string>
   </property>
  </widget>
  <widget class="QLabel" name="image9">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Image 9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="name_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50)</string>
   </property>
   <property name="text">
    <string>ItemName1</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="name_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50)</string>
   </property>
   <property name="text">
    <string>ItemName2</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="name_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50)</string>
   </property>
   <property name="text">
    <string>ItemName3</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="name_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50)</string>
   </property>
   <property name="text">
    <string>ItemName4</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="name_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50)</string>
   </property>
   <property name="text">
    <string>ItemName5</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="name_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50)</string>
   </property>
   <property name="text">
    <string>ItemName6</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="name_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50)</string>
   </property>
   <property name="text">
    <string>ItemName7</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="name_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>520</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50)</string>
   </property>
   <property name="text">
    <string>ItemName8</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="name_9">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50)</string>
   </property>
   <property name="text">
    <string>ItemName9</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>560</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previous">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>560</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>560</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to List</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>20</width>
     <height>431</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>20</width>
     <height>431</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
